core/chains/evm/txmgr/attempts.go

```diff
@@ -98,15 +98,15 @@ func (c *evmTxAttemptBuilder) NewCustomTxAttempt(etx EvmTx, fee gas.EvmFee, gasL
 }
 
 // NewEmptyTxAttempt is used in ForceRebroadcast to create a signed tx with zero value sent to the zero address
-func (c *evmTxAttemptBuilder) NewEmptyTxAttempt(nonce uint64, feeLimit uint32, fee gas.EvmFee, fromAddress evmtypes.Address) (attempt EvmTxAttempt, err error) {
+func (c *evmTxAttemptBuilder) NewEmptyTxAttempt(nonce evmtypes.Nonce, feeLimit uint32, fee gas.EvmFee, fromAddress evmtypes.Address) (attempt EvmTxAttempt, err error) {
 	value := big.NewInt(0)
 	payload := []byte{}
 
 	if fee.Legacy == nil {
 		return attempt, errors.New("NewEmptyTranscation: legacy fee cannot be nil")
 	}
 
-	tx := types.NewTransaction(nonce, fromAddress.Address, value, uint64(feeLimit), fee.Legacy.ToInt(), payload)
+	tx := types.NewTransaction(uint64(nonce.N), fromAddress.Address, value, uint64(feeLimit), fee.Legacy.ToInt(), payload)
```

Comment:
+1 to Jordan's comment about using a helper - it's cleaner than pulling the internal type and then wrapping it to ensure the types line up